qutebrowser/config/configdata.yml

```diff
@@ -2324,12 +2329,12 @@ bindings.default:
     Default keybindings. If you want to add bindings, modify
     `bindings.commands` instead.
 
-    The main purpose of this setting is that you can set it to an empty dictionary if you
-    want to load no default keybindings at all.
+    Main purpose of this setting is that you can set it to an empty
```

Comment:
I think that dropping the `The` in this case and the case below sounds a bit weird as well. Maybe we can make an exeption for longer blocks like this one, since they feel more like a dialouge.

Reply:
I agree. This was unintended and I tried to undo those replacements (there where a few more like 'The following ... are supported'). I must have missed this one.